lib/app.js

```diff
@@ -83,7 +83,11 @@ class Container extends Component {
       // https://github.com/gaearon/react-hot-loader/issues/442
       return (
         <AppContainer warnings={false} errorReporter={ErrorDebug}>
-          <Component {...props} url={url} />
+          {
+            Component.name === 'render'
```

Comment:
On other idea to wrap the `Component` inside a function if it's a React component class.